ignition-fuel_tools-4/PKGBUILD

```diff
@@ -14,9 +14,9 @@ makedepends=('ignition-cmake>=2')
 optdepends=('ruby')
 conflicts=()
 source=("${pkgname}-${pkgver}.tar.gz::https://github.com/ignitionrobotics/ign-fuel-tools/archive/ignition-fuel-tools4_${pkgver}.tar.gz")
```

Comment:
This should also be changed to `gazebosim/gz-fuel-tools` i think
and the `url` field.

Reply:
Done!

Reply:
The `url` field still needs to be changed

Reply:
So it does.